src/postgres/tree/PostgresDatabaseTreeItem.ts

```diff
@@ -94,8 +92,8 @@ export class PostgresDatabaseTreeItem extends AzureParentTreeItem<ISubscriptionC
     }
 
     public async deleteTreeItemImpl(): Promise<void> {
-        const client: PostgreSQLManagementClient = createAzureClient(this.root, PostgreSQLManagementClient);
-        await client.databases.deleteMethod(azureUtils.getResourceGroupFromId(this.fullId), this.parent.name, this.databaseName);
+        const config = await getClientConfig(this.parent, 'postgres');
+        const pool = new Pool(config);
```

Comment:
There doesn't seem to be any difference for this use case. I was having some problems earlier with client.query, I realize it was because I was not closing the connection. runPostgresQuery works for this scenario as well.